app/graphql/types/rule.rb

```diff
@@ -9,6 +9,7 @@ class Rule < Types::BaseObject
     field :id, ID, null: false
     field :title, String, null: false
     field :ref_id, String, null: false
+    field :remediation_issue_id, String, null: true
```

Comment:
GraphQL might be slower after this. We might consider adding a preload of related items, if anyone knows the recommended way to do that.